x-pack/plugins/security_solution/public/common/components/event_details/alert_summary_view.test.tsx

```diff
@@ -255,12 +256,39 @@ describe('AlertSummaryView', () => {
       {
         category: 'kibana',
         field: 'kibana.alert.threshold_result.terms',
-        values: ['{"field":"host.name","value":"Host-i120rdnmnw"}'],
-        originalValue: ['{"field":"host.name","value":"Host-i120rdnmnw"}'],
+        // value and originalValue don't matter here since the data comes from `_source` in the end
+        values: [''],
+        originalValue: [''],
+      },
+      {
+        category: 'kibana',
+        field: 'kibana.alert.threshold_result.cardinality',
+        // value and originalValue don't matter here since the data comes from `_source` in the end
+        values: [''],
+        originalValue: [''],
       },
     ] as TimelineEventsDetailsItem[];
     const renderProps = {
       ...props,
+      rawEventData: {
+        _source: {
+          'kibana.alert.threshold_result': {
```

Comment:
Is it a requirement for this feature to be backwards compatible with alert fields prefixed with `signal` instead of `kibana.alert`? If so, consider adding a test case for that.